tests/subtests/test_cardano.rs

```diff
@@ -169,6 +169,55 @@ pub async fn test(bitbox: &PairedBitBox) {
             "ed0d6426efcae3b02b963db0997845ba43ed53c131aa2f0faa01976ddcdb3751",
         );
     }
+    // Delegating vote to a drep
+    {
+        let transaction = pb::CardanoSignTransactionRequest {
+            network: pb::CardanoNetwork::CardanoMainnet as i32,
+            inputs: vec![pb::cardano_sign_transaction_request::Input {
+                keypath: keypath_input.to_vec(),
+                prev_out_hash: hex::decode(
+                    "59864ee73ca5d91098a32b3ce9811bac1996dcbaefa6b6247dcaafb5779c2538",
+                )
+                    .unwrap(),
+                prev_out_index: 0,
+            }],
+            outputs: vec![pb::cardano_sign_transaction_request::Output {
+                encoded_address: change_address.clone(),
+                value: 2741512,
+                script_config: Some(change_config.clone()),
+                ..Default::default()
+            }],
+            fee: 191681,
+            ttl: 41539125,
+            certificates: vec![
+                pb::cardano_sign_transaction_request::Certificate {
+                    cert: Some(
+                        pb::cardano_sign_transaction_request::certificate::Cert::VoteDelegation(
+                            pb::cardano_sign_transaction_request::certificate::VoteDelegation {
+                                keypath: vec![2147485500, 2147485463, 2147483648, 2, 0],
+                                r#type: pb::cardano_sign_transaction_request::certificate::vote_delegation::CardanoDRepType::AlwaysAbstain.into(),
+                                drep_credhash: None,
+                            },
+                        ),
+                    ),
+                },
+            ],
+            withdrawals: vec![],
+            validity_interval_start: 41110811,
+            allow_zero_ttl: false,
+        };
+
+        let witness = bitbox.cardano_sign_transaction(transaction).await.unwrap();
```

Comment:
I think you'll need to check for success if the firmware version is >= 9.21.0, and for failure otherwise, as older firmwares don't yet have support for vote delegation.